oak_containers_launcher/src/qemu.rs

```diff
@@ -105,6 +105,16 @@ impl Qemu {
         }
         cmd.args(["-netdev", "user,id=netdev"]);
         cmd.args(["-device", "virtio-net,netdev=netdev"]);
+        // Set up the virtio-vsock device, as it is used by the example app.
+        // TODO(#709): Remove this and use networking for there as well.
+        cmd.args([
+            "-device",
+            format!(
+                "vhost-vsock-pci,id=vhost-vsock-pci0,guest-cid={}",
+                std::process::id()
+            )
+            .as_str(),
+        ]);
```

Comment:
was removed in https://github.com/project-oak/oak/pull/4149, but is still needed for the current example app